test/factories/users.rb

```diff
@@ -17,6 +17,10 @@
       end
```

Comment:
Should we also update the `adopter` factory to create the associated `form_submission` after we add the role? See line 30 of this file.

Reply:
Yeah we can do that